guides/cancelOrders.md

```diff
@@ -4,52 +4,78 @@
 
 ## Cancel orders using order nonces
 
+This method is used to invalidate the `orderNonce`. If multiple maker orders share the same nonce, they will all be cancelled and non-executable. An `orderNonce` is also invalidated once an order with that nonce is executed.
+
 ```ts
 import { LooksRare, SupportedChainId } from "@looksrare/sdk-v2";
 
 const lr = new LooksRare(SupportedChainId.MAINNET, provider, signer);
 
 // Cancel order nonce 0
 const tx = await lr.cancelOrders([0]).call();
+
+// Wait for the transaction to be processed
```

Comment:
These comments are unnecessary because it's considered a basic knowledge to know what a `tx.wait()` does when you use ethers